### PR TITLE
testing/py3-pydocstyle: new aport

### PR DESCRIPTION
**Pull request by russkel**

### CODE REVIEW

File: testing/py3-pydocstyle/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+source="$_pkgname-$pkgver.tar.gz::https://github.com/PyCQA/$_pkgname/archive/$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	# integration tests call pip which doesn't exist
+    PYTHONPATH="$builddir/build/lib" py.test --deselect src/tests/test_integration.py -vv src/tests
```
Comment (Cogitri) [inline testing/py3-pydocstyle/APKBUILD]: fwiw you've accidentally used spaces instead of tabs here.

File: testing/py3-pydocstyle/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+pkgname="py3-pydocstyle"
+_pkgname="pydocstyle"
+pkgver="4.0.1"
+pkgrel=0
+pkgdesc="Static analysis tool for checking compliance with Python docstring conventions"
+url="https://pydocstyle.org/"
+arch="noarch"
+license="MIT"
+depends="python3 py3-snowballstemmer"
+makedepends="python3 py3-pytest py3-mock py3-six"
```
Comment (Cogitri) [inline testing/py3-pydocstyle/APKBUILD]: depends will be installed during build too, so adding python3 to both makedepends and depends does nothing (just having it in depends is sufficient)

Comment (Cogitri) [reply testing/py3-pydocstyle/APKBUILD]: This should most likely have py3-setuptools in its makedepends though

File: testing/py3-pydocstyle/APKBUILD (hunk 1 above)
Comment (russkel) [reply testing/py3-pydocstyle/APKBUILD]: Oops. need to fix my editor to show this.

**Comment by russkel** [comment]
Thanks @Cogitri. Fixed as requested.

**Pull request closed without merging**